spec/controllers/users/phone_setup_controller_spec.rb

```diff
@@ -27,10 +27,10 @@
       expect(@analytics).to receive(:track_event).
         with(Analytics::MULTI_FACTOR_AUTH_PHONE_SETUP, result)
 
-      patch :set, params: {
+      patch :create, params: {
         user_phone_form: {
           phone: '703-555-010',
-          otp_delivery_preference: :sms,
+          # otp_delivery_preference: :sms,
```

Comment:
We should remove this line instead of commenting it out. Same for the other places this is commented out.